homeassistant/components/sensor/co2mini.py

```diff
@@ -0,0 +1,107 @@
+"""
+Support for the CO2 and temperature sensor of CO2meter's CO2Mini USB indoor air
+quality monitor and compatible devices.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.co2mini/
+"""
+
+import asyncio
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_DEVICE, CONF_FRIENDLY_NAME, STATE_UNKNOWN
+from homeassistant.core import callback
+from homeassistant.helpers.entity import Entity
+import homeassistant.helpers.config_validation as cv
+
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['https://github.com/jannau/CO2Meter/archive/fea592cfd2b54124e4afdc20320f5c929e1c07d6.zip#CO2Meter==2.0']
```

Comment:
sure, I just wanted to make the it public and the code depended at the time I wrote it on a change only in my tree. I'll see to get the dependency on PyPI before removing the WIP and update the pull request